### PR TITLE
fix nock count and search assert

### PR DESCRIPTION
**Pull request by eddiemonge**
duplicate of #265 but less code

### CODE REVIEW

File: test/route-install.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('install route', function () {
           .reply(200, { rows: this.rows })
         .filteringPath(/\/registry\/[^\?]+$/g, '/registry/pkg')
           .get('/registry/pkg')
-          .times(2)
+          .times(3)
```
Comment (eddiemonge) [inline test/route-install.js]: nock needed one more time to get it right

File: test/route-install.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ describe('install route', function () {
           var choices = arg[0].choices;
           assert.equal(_.where(choices, { value: 'generator-foo' }).length, 1);
           assert.equal(_.where(choices, { value: 'generator-unicorn-1' }).length, 1);
-          assert.equal(_.where(choices, { value: 'generator-unicorn' }).length, 0);
+          assert.equal(_.where(choices, { value: 'generator-unicorn' }).length, 1);
```
Comment (eddiemonge) [inline test/route-install.js]: unicorn is in the name so should match

Comment (SBoudrias) [inline test/route-install.js]: It shouldn't match as generator-unicorn is already installed. See line 28

**Comment by gaboesquivel** [comment]
:+1:

**Pull request closed without merging**